schemas/livePaper.schema.tpl.json

```diff
@@ -0,0 +1,38 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/LivePaper",
+  "_extends": "/core/products/researchProduct.schema.tpl.json",
+  "required": ["author"],
+  "properties": {
+    "author": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add one or several authors (person or organization) that contributed to the production and publication of this live paper.",
+      "_linkedCategories": ["legalPerson"]
+    },
+    "digitalIdentifier": {
+      "_instruction": "Add the globally unique and persistent digital identifier of this research product. Note that this digital identifier will be used to reference all attached research product versions.",
+      "_linkedTypes": ["https://openminds.ebrains.eu/core/DOI"]
+    },
+    "hasVersion": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add one or several versions of this live paper.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/publications/LivePaperVersion"
+      ]
+    },
+    "relatedPublication": {
```

Comment:
this is currently only reserved for ResearchProducVersions. If we want this for the ResearchProducts we can add this centrally in core.

Reply:
I've moved it to LivePaperVersion